src/80211_if.c

```diff
@@ -2941,6 +2974,13 @@ static int uccp420wlan_init(void)
 		}
 	}
 
+	error=start_core(hw);
```

Comment:
Instead of making changes in the core code, can't we just move the fw_loading  call here from start()?